Clarify data classification and scope for data sharing tools (#1090)
FredHutch/wiki: _scicomputing/store_collaboration.md

primary_reviewers: vortexing
---

> Before transferring data, please make sure you have appropriate protections in place- make sure you are allowed to share data via your desired mechanism, that agreements are in place or won't be violated by transferring data.  Contact `dataprotection@fredhutch.org` if you have questions or need assistance evaluating your transfer needs.

Reviewer: ```suggestion
Before transferring data, please make sure you have appropriate protections in place and are allowed to share data via your desired mechanism. Contact `dataprotection@fredhutch.org` if you have questions or need assistance evaluating your transfer needs.
```
